### PR TITLE
Added Pennsylvania statewide 2016/17 precincts 

### PR DESCRIPTION
**Pull request by migurski**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -778,10 +778,10 @@ out/41-oregon/state.gpkg: data/41-oregon/metro-portland/2016/precinct.zip data/4
 
 out/42-pennsylvania/state.gpkg: data/42-pennsylvania/statewide/2011/2011-Voting-District-Boundary-Shapefiles.zip
 	mkdir -p out/42-pennsylvania
-	unzip -d out/42-pennsylvania data/42-pennsylvania/statewide/2011/2011-Voting-District-Boundary-Shapefiles.zip
-	ogr2ogr -sql "SELECT '2011' AS year, 'Pennsylvania' AS state, COUNTYFP11 AS county, NAMELSAD AS precinct, 'polygon' AS accuracy FROM VTDS" \
```
Comment (nvkelso) [inline Makefile]: Why not `2017` here instead of `2016`?

Comment (migurski) [reply Makefile]: This source is a little bit of a chimera: it was explained to me as "2017 precincts with 2016 vote totals," which to me means 2016 precincts but I guess PA’s counties are continuously messing with boundaries?

**Comment by migurski** [comment]
CI seems to be failing on another state altogether:

```
unzip -d out/37-north-carolina/source data/37-north-carolina/statewide/2016/SBE_PRECINCTS_20160826.zip
Archive:  data/37-north-carolina/statewide/2016/SBE_PRECINCTS_20160826.zip
  inflating: out/37-north-carolina/source/Precincts.dbf  
  inflating: out/37-north-carolina/source/Precincts.prj  
  inflating: out/37-north-carolina/source/Precincts.shp  
  inflating: out/37-north-carolina/source/Precincts.shp.xml  
  inflating: out/37-north-carolina/source/Precincts.shx  
ogr2ogr -sql "SELECT '2016' AS year, '37' AS state, COUNTY_ID AS county, CONCAT('37', CAST(COUNTY_ID as character(20)), PREC_ID) AS precinct, 'polygon' AS accuracy FROM precincts" \
		-s_srs '+proj=lcc +lat_1=34.33333333333334 +lat_2=36.16666666666666 +lat_0=33.75 +lon_0=-79 +x_0=609601.2199999997 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
		-t_srs EPSG:4326 -overwrite -f GPKG out/37-north-carolina/state.gpkg out/37-north-carolina/source/precincts.shp
FAILURE:
Unable to open datasource `out/37-north-carolina/source/precincts.shp' with the following drivers.
  -> PCIDSK
  -> netCDF
  -> JP2OpenJPEG
  -> PDF
  -> ESRI Shapefile
etc.
```

**Comment by nvkelso** [comment]
Is it fussy about the p and P capitalization difference, or something else?

**Comment by migurski** [comment]
Under Linux with a case-sensitive file system, it would be fussy about capitalization yeah.

**Comment by nvkelso** [comment]
Hmm, CI didn't run again, maybe because PR was already approved? At any rate... I'm slowly setting up new MacBook and will confirm the PR manually and then merge.

**Comment by migurski** [comment]
Cool — I can try this as well, I think I still have this monster checkout someplace useful. :)

**Comment by migurski** [comment]
Heh. The CI for this monster repo is me manually running shit under Docker. I’ll be able to do it but possible not this evening.

**Comment by migurski** [comment]
Preview for commit cf5054c6e9fb93422c557ecea7e52db1c4ace266:

**Comment by migurski** [comment]
CI is back to working!

**Comment by nvkelso** [comment]
Woot!

**Pull request merged**